### ISSUE TITLE
Add version detail in Help/About dialog and/or in Log file

### ISSUE THREAD

**Issue by EdoFro** [freeplane/freeplane#837]
I think it would be nice/useful to have an easy way to get which is the instaled version from Freeplane (**including if it's a prerelease version and its number**)
For example: `v1.10.6_01` or  `v1.10.6 pre 01`

for example, when clicking **Help / About**, it shows:
![image](https://user-images.githubusercontent.com/35700575/203074594-5d00903d-5bfd-4123-8308-918746dc2d79.png)

with no information if this is a prerelease version (which one?) or a stable version. For example: `v1.10.6_01` or  `v1.10.6 pre 01`

The Log file doesn't have this information neither:

```ini
STDOUT: Nov 20, 2022 6:22:57 PM org.freeplane.core.util.LogUtils info
INFO: freeplane_version = 1.10.6; freeplane_xml_version = freeplane 1.9.13
git revision = 8ebd14db413c41c13d4362eb9d78c92adc926c87 Dimitry Polivaev 2022-11-18T19:57:53Z
java_version = 15.0.2; os_name = Windows 10; os_version = 10.0
STDOUT: Nov 20, 2022 6:22:57 PM org.freeplane.core.util.LogUtils info
INFO: added parsing support for standard date format dd-MM-yy
...

```

#### Request:
Add Freeplane's version detail in **Help/About** dialog and/or in **Log file**

Thanks in advance,
edo

**Comment by euu2021** [comment]
Yes, please. I need to search in my Downloads folder to find which was the latest version that I downloaded, so I can know the version that I'm using 😅

**Comment by EdoFro** [comment]
I also tried

```groovy
c.freeplaneVersion
```

to get the version's detail, and I got these properties values:

![image](https://user-images.githubusercontent.com/35700575/206511145-0d648d7b-3458-4936-934e-2a21f2c9690a.png)

I'm using `v1.10.6 pre03`, but the "**pre**" and "**03**" part are nowhere to find as properties values.

Maybe they are not updated in the release process. It's that possible?

**Comment by EdoFro** [comment]
if I execute:
```groovy
def v = c.freeplaneVersion.getVersion('v1.10.6 pre 03')
```

the properties of variable "v" are:
![image](https://user-images.githubusercontent.com/35700575/206517537-a0462c6e-26fd-4a2c-9fb7-3287993e9be9.png)

and `v.toString()` returns  `'1.10.6 pre 3'`

**Comment by EdoFro** [comment]
I revisited this idea yesterday.

## Notes to my future self (or for anybody who wants to implement this feature):

(I was not at my computer these days, so I couldn't test this)

This is how I think this feature may be implemented:

### 1. Add a new property to **version.properties**

file:  [freeplane/src/viewer/resources/version.properties](https://github.com/freeplane/freeplane/blob/dafd4fe203e8b911f09ad7e574e4760de79f5baf/freeplane/src/viewer/resources/version.properties)

add property: `freeplane_detailed_version=no detailed version available`

This property should be changed to the real value by the Gradle building process.

------

### 2. Change FreeplaneVersion.java to access this new property

file: [freeplane/src/main/java/org/freeplane/core/util/FreeplaneVersion.java](https://github.com/freeplane/freeplane/blob/dafd4fe203e8b911f09ad7e574e4760de79f5baf/freeplane/src/main/java/org/freeplane/core/util/FreeplaneVersion.java)

add two new fields and change one:
```java
	private static final FreeplaneVersion VERSION = FreeplaneVersion.loadVersion(false);
	private static final FreeplaneVersion DETAILED_VERSION = FreeplaneVersion.loadVersion(true);

	public static final String DETAILED_VERSION_KEY = "freeplane_detailed_version";
```

add a new method
```java
        public static FreeplaneVersion getDetailedVersion() {
                return DETAILED_VERSION;
        }
```

change method [loadVersion()](https://github.com/freeplane/freeplane/blob/dafd4fe203e8b911f09ad7e574e4760de79f5baf/freeplane/src/main/java/org/freeplane/core/util/FreeplaneVersion.java#L82)  to something like this:

```java
        private static FreeplaneVersion loadVersion(boolean useDetailedVersion) {
                final String versionKey = useDetailedVersion ? DETAILED_VERSION_KEY : VERSION_KEY;
                final Properties versionProperties = FileUtils.loadProperties(VERSION_PROPERTIES);
                final String versionString = versionProperties.getProperty(versionKey );
                final String versionStatus = versionProperties.getProperty("freeplane_version_status");
                final FreeplaneVersion version = FreeplaneVersion.getVersion(versionString);
                if (versionStatus != "")
                        version.mType = versionStatus;
                return version;
        }

```

------

### 3. Changes in AboutAction.java to show the detailed version

file: [freeplane/src/main/java/org/freeplane/features/help/AboutAction.java](https://github.com/freeplane/freeplane/blob/dafd4fe203e8b911f09ad7e574e4760de79f5baf/freeplane/src/main/java/org/freeplane/features/help/AboutAction.java)

modify method [actionPerformed()](https://github.com/freeplane/freeplane/blob/dafd4fe203e8b911f09ad7e574e4760de79f5baf/freeplane/src/main/java/org/freeplane/features/help/AboutAction.java#L60) to show detailed version
- additional to normal version
- as a text under the git id line

------

### 4. other changes

1. [ApplicationResourceController.showSysInfo()](https://github.com/freeplane/freeplane/blob/dafd4fe203e8b911f09ad7e574e4760de79f5baf/freeplane/src/main/java/org/freeplane/main/application/ApplicationResourceController.java#L64)
2. [ReportGenerator.createInfo()](https://github.com/freeplane/freeplane/blob/dafd4fe203e8b911f09ad7e574e4760de79f5baf/freeplane_plugin_bugreport/src/main/java/org/freeplane/plugin/bugreport/ReportGenerator.java#L151)

-----

### 5. Changes in Build.Gradle

file: [build.gradle](https://github.com/freeplane/freeplane/blob/dafd4fe203e8b911f09ad7e574e4760de79f5baf/build.gradle)

add code to:
1. put detailed version into **props**
something like: `props['freeplane_detailed_version'] = project.ext.distVersion`
2. save **props** into `rootDir.path + '/freeplane/src/viewer/resources/version.properties'` file

-----

I hope it helps,

edo
